custom-elements/ElementInternals-focusBehavior.html

```diff
@@ -0,0 +1,70 @@
+<!DOCTYPE html>
+<link rel="help" content="https://html.spec.whatwg.org/C/#dom-elementinternals-focusbehavior">
```

Comment:
@tkent-google @domenic pardon my ignorance but is there a reason why this help link is broken?

Other than that Looks GREAT To Me.

Reply:
This is for a spec PR that is not yet merged. Once the spec PR is merged the link will start working.